ClimsoftVer4/ClimsoftVer4/ucrFormDaily2.vb

```diff
@@ -256,33 +256,39 @@ Public Class ucrFormDaily2
         Next
     End Sub
     ''' <summary>
-    ''' Checks if the computer computed total is same as the user total
-    ''' If not gives a warning
+    ''' Checks if total for current element is required
+    ''' Checks if the computed total is same as the user entered total.
     ''' </summary>
     Public Sub checkTotal()
-        'Check total if required
-        ' Am not sure how to get this value yet
-        'If totalRequired = 1 Then
-
+        'Check total if required from obselements table from qcTotalRequired field
+        Dim clsDataDefinition As DataCall
+        Dim dtbl As DataTable
         Dim elemTotal As Integer = 0
         Dim expectedTotal As Integer
         Dim ctr As Control
         Dim ctrVFP As New ucrValueFlagPeriod
 
+        clsDataDefinition = New DataCall
```

Comment:
@Patowhiz I have edited this so that it checks if total is required from the obselement table and if so, then the user needs to put correct totals before proceedings.  You could edit yours to work in same way in hourly wind. 
Thanks.

Reply:
@maxwellfundi in samuel form for hourlywind, he is checking if the computer computed total is same as the user total by just summing up the entire elements and checking if it's equal to the user total instead of going to the obselement. Should I still proceed with implementing this way?

Reply:
Sorry, I already had implemented this in ucrHourlyWind. You can check it. Though I've impemented it a little bit differently we can discuss on that.